reftests/local.ron

```diff
@@ -45,4 +45,26 @@
 			expect: Buffer("buffer.output", [1, 0, 0, 0]),
 		),
 	},
+	"clear": {
+		"fill-whole": (
+			features: (bits: 0),
+			jobs: ["fill-whole"],
+			expect: Buffer("buffer.1", [0, 0, 255, 0, 0, 0, 255, 0]),
```

Comment:
why do you need different buffers for those tests?

Reply:
I wasn't sure whether it was safe to reuse the same writable resource (whether it was recreated per test or whether two tests could write to it at the same time). Assuming it's safe I'll collapse them into a single resource instead.